stable/jenkins/README.md

```diff
@@ -163,6 +163,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `master.prometheus.alertingrules` | Array of prometheus alerting rules | `[]`                                        |
 | `master.prometheus.alertingRulesAdditionalLabels` | Additional labels to add to the prometheus rule object     | `{}`                                   |
 | `master.priorityClassName`        | The name of a `priorityClass` to apply to the master pod | Not set               |
+| `master.test.enabled`             | Enable master UI test during deployment | `true`                                 |
```

Comment:
Should we rename the description to something like "Can be used to disable rendering test resources when using helm template"?

Reply:
@torstenwalter Apologies for the delay. I pushed the changes for you to review.